internal/config/config_type.go

```diff
@@ -16,6 +16,10 @@ const (
 	PromptsEnabled     = "enabled"
 )
 
+var configValues = map[string][]string{
+	"git_protocol": {"ssh", "https"},
```

Comment:
Hmm, we could also add
```go
"prompt": {"enabled", "disabled"}
```

Reply:
I think this should probably go in a follow up PR 👍